frontend/containers/VaultsContainer.vue

```diff
@@ -37,7 +45,16 @@ export default Vue.extend({
         ...mapGetters('wallet', {
             isWalletLoading: 'isLoading',
             walletAddress: 'getAddress',
+            walletBalances: 'walletBalances',
+            isWithdrawing: 'isDepositingOrWithdrawing',
         }),
+        ...mapGetters('authorizations', {
+            isAuthorizing: 'isWalletAuthorizationLoading',
+            isWalletAuthorized: 'isWalletAuthorizationDone',
+        }),
+        daiVatBalance(): BigNumber | undefined {
```

Comment:
Why do we have this extra? Can't you just pass `walletbalances?.walletVATDAI` directly in the component above?